### PR TITLE
XW-3612 | Language selector visible on all pages that have interlanguage links

### PR DESCRIPTION
**Pull request by latata**
https://wikia-inc.atlassian.net/browse/XW-3612
@Wikia/x-wing

### CODE REVIEW

File: extensions/wikia/LocalSitemapPage/LocalSitemapPage.setup.php [hunk 1]
```diff
@@ -12,4 +12,3 @@
 // hooks
 $wgHooks['ArticleFromTitle'][] = 'LocalSitemapPageHooks::onArticleFromTitle';
 $wgHooks['PageHeaderActionButtonShouldDisplay'][] = 'LocalSitemapPageHooks::onPageHeaderActionButtonShouldDisplay';
-$wgHooks['PageHeaderLanguageSelectorShouldDisplay'][] = 'LocalSitemapPageHooks::onPageHeaderLanguageSelectorShouldDisplay';
```
Comment (rybmat) [inline extensions/wikia/LocalSitemapPage/LocalSitemapPage.setup.php:15]: why did you removed this hook? it was on purpose to not display the language on local sitemap page

**Review by rybmat — CHANGES_REQUESTED** [review]

File: extensions/wikia/LocalSitemapPage/LocalSitemapPage.setup.php (hunk 1 above)
Comment (latata) [reply extensions/wikia/LocalSitemapPage/LocalSitemapPage.setup.php:15]: Currently it's not necessary as we only show language selector for pages where `count( $this->languageList ) > 0` is `true`. On local sitemap it's true.

Comment (rybmat) [reply extensions/wikia/LocalSitemapPage/LocalSitemapPage.setup.php:15]: 👍

**Review by rybmat — APPROVED** [review]

File: extensions/wikia/PageHeader/Languages.class.php [hunk 2]
```diff
@@ -21,10 +21,7 @@ public function __construct( WikiaApp $app ) {
 		$this->title = \RequestContext::getMain()->getTitle();
 		$this->currentLangName = \Language::getLanguageName( $this->title->getPageLanguage()->getCode() );
 		$this->languageList = $this->handleLanguages( $app );
-
-		$shouldDisplay = $this->title->isContentPage();
-		wfRunHooks( 'PageHeaderLanguageSelectorShouldDisplay', [$this->title, &$shouldDisplay] );
-		$this->shouldDisplay = $shouldDisplay;
+		$this->shouldDisplay = count( $this->languageList ) > 0;
```
Comment (rogatty) [inline extensions/wikia/PageHeader/Languages.class.php]: This plus:
```
 	public function shouldDisplay(): bool {
 		return $this->shouldDisplay && count( $this->languageList ) > 0;
 	}
```
don't make sense together.

Comment (rybmat) [reply extensions/wikia/PageHeader/Languages.class.php]: heh, true, didn't notice that. @rogatty it is fixed now

**Review by weronika — APPROVED** [review]

**Pull request merged**